Sources/StreamChatSwiftUI/ChatChannelList/ChatChannelList.swift

```diff
@@ -56,13 +58,24 @@ public struct ChannelList<Factory: ViewFactory>: View {
         self.scrollable = scrollable
         _selectedChannel = selectedChannel
         _swipedChannelId = swipedChannelId
+        _offsetY = offsetY
     }
     
     public var body: some View {
         Group {
             if scrollable {
                 ScrollView {
-                    channelsVStack
+                    ZStack {
```

Comment:
Thanks for the PR. Not sure if this should be enabled by default for all users. We should probably have a config with an opt-in. Will think about this and do a follow-up.